Build/Classes/Composer/InstallSitePackage.php

```diff
@@ -0,0 +1,77 @@
+<?php
+namespace Neos\BaseDistribution\Composer;
+
+use Composer\Console\Application;
+use Composer\Script\Event;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Neos\Splash\DistributionBuilder\Service\PackageService;
+use Neos\Splash\DistributionBuilder\Service\JsonFileService;
+use Neos\Splash\DistributionBuilder\Domain\ValueObjects\PackageRequirement;
+
+/**
+ * A comopser post-create-project script to allow a choice of adding the demo site (and possible other things in the future)
+ */
+class InstallSitePackage
+{
+    /**
+     * Setup the neos distribution
+     *
+     * @param Event $event
+     * @throws \Neos\Utility\Exception\FilesException
+     * @throws \Exception
+     */
+    public static function setupDistribution(Event $event)
+    {
+        $distributionReadyMessagesBase = [
+            '',
+            'Your Neos was prepared successfully.',
+            '',
+            'For local development you still have to:',
+            '1. Add database credentials to Configuration/Development/Settings.yaml (or start the setup by calling the /setup url)',
+            '2. Migrate database "./flow doctrine:migrate"',
+        ];
+
+        $io = $event->getIO();
+        $io->write([
+            '',
+            'Welcome to Neos',
+            ''
+        ]);
+
+        if (!$io->isInteractive()) {
+            $io->write('Non-Interacctive installation, installing no additional package(s).');
+            $io->write($distributionReadyMessagesBase);
+            return;
+        }
+
+        $choices = [
+            'start with the Neos Demo content',
+            'empty Neos'
```

Comment:
Just a mini suggestion:
Wouldn't it make more sense to have `empty Neos` the first (i.e. default) option since there will be more example content in the future probably.
Also is "empty Neos" a good name? It installs some packages that are not required per se (like `neos/seo` and `neos/redirecthandler-*`). I would totally love some kind of "bare bones" installation, too, that only requires the absolutely required packages

Reply:
..I guess it was an intentional decision to keep the current behavior the default, makes sense I guess! But it could be the default and not be the first option, too